ietf/name/fixtures/names.json

```diff
@@ -5998,10 +5999,24 @@
     "model": "name.constraintname",
     "pk": "bethere"
   },
+  {
+    "fields": {
+      "desc": "Indicates other WGs the chairs also lead or will be active participants in",
+      "editor_label": "<i class=\"fa fa-gavel\"></i>",
+      "is_group_conflict": true,
+      "name": "Chair conflict",
+      "order": 8,
+      "penalty": 100000,
+      "used": true
+    },
+    "model": "name.constraintname",
+    "pk": "chair_conflict"
+  },
```

Comment:
I suspect that this block and other following blocks about group conflict are probably not meant to be here.  Right?